_std/macros/flock.dm

```diff
@@ -1,4 +1,22 @@
-// hi here's some flockdrone BS - cirr
+// type check macros
 #define isfeathertile(x) (istype(x, /turf/simulated/floor/feather) || istype(x, /turf/simulated/wall/auto/feather))
 #define isflock(x) (istype(x, /mob/living/intangible/flock) || istype(x, /mob/living/critter/flock))
```

Comment:
Ok I know this will be an obnoxious thing to replace BUT reading `isflock` makes me assume that this will check for flock structures, flock walls, flock mobs, etc
so it **just** checking flock mobs is... confusing
(i had to scroll up here while checking the conveyors file to see what this macro was actually doing)
Would appreciate a rename to `isflockmob` or something